manifests/params.pp

```diff
@@ -74,12 +74,23 @@
       $site_packages = "/usr/local/lib/python${version}/dist-packages"
     }
     redhat: {
+      case $::operatingsystemmajrelease {
+        '6': {
+          $version       = '2.6'
+          $pip           = 'python-pip'
+        }
+        '7': {
+          $version       = '2.7'
+          $pip           = 'python2-pip'
+        }
+        default: {
+          fail("redhat version \"${::operatingsystemmajrelease}\" not supported")
```

Comment:
I don't know RedHat too well -- is it safe to assume versions less than 6 won't support `pip`?